fisher.fish

```diff
@@ -815,27 +823,43 @@ function __fisher_plugin_enable -a path
 
     for file in $path/{functions/,}*.{py,awk}
         set -l base (basename "$file")
-        command ln -sf "$file" "$fish_path/functions/$base"
+        if eval $fisher_copy
```

Comment:
@skisulli `eval`?

Reply:
I'm new to fish and then I had just `if $fisher_copy` the shell errored with the following message.

```
~/.config/fish/functions/fisher.fish (line 35): Variables may not be used as commands. In fish, please define a function or use 'eval $fisher_copy'.
        if $fisher_copy
           ^
in function '__fisher_plugin_enable'
	called on line 449 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/rsullivan/.config/fisherman/nvm'
```

What would be the proper way to check the value of a boolean?

Reply:
I'm new to fish and then I had just `if $fisher_copy` the shell errored with the following message.

```
~/.config/fish/functions/fisher.fish (line 35): Variables may not be used as commands. In fish, please define a function or use 'eval $fisher_copy'.
        if $fisher_copy
           ^
in function '__fisher_plugin_enable'
	called on line 449 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/rsullivan/.config/fisherman/nvm'
```

What would be the proper way to check the value of a boolean?

Reply:
I'm new to fish and then I had just `if $fisher_copy` the shell errored with the following message.

```
~/.config/fish/functions/fisher.fish (line 35): Variables may not be used as commands. In fish, please define a function or use 'eval $fisher_copy'.
        if $fisher_copy
           ^
in function '__fisher_plugin_enable'
	called on line 449 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/rsullivan/.config/fisherman/nvm'
```

What would be the proper way to check the value of a boolean?

Reply:
```if test $fisher_copy = true```
or
```if test $fisher_copy = false```

Reply:
removed the eval calls